### PR TITLE
Create dblp.json

### PR DESCRIPTION
**Pull request by grolimur**
DBLP is a computer science centered bibliographic database

### CODE REVIEW

**Comment by grolimur** [comment]
@egh I have a few more databases in mind. Do you prefer me to add them one by one, so that you can validate/reject each one? Or do you prefer me to add a bunch of them in the same commit?

File: engines_json/dblp.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "title": "DBLP",
+  "_icon": "https://dblp.org/img/dblp.icon.192x192.png",
```
Comment (egh) [inline engines_json/dblp.json:3]: This should be `icon`

File: engines_json/dblp.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+  "_urlTemplate": "https://dblp.uni-trier.de/search?q={z:title}",
```
Comment (egh) [inline engines_json/dblp.json:4]: This should be `linktemplate`.

**Comment by egh** [comment]
All at once is fine, thanks!

File: engines_json/dblp.json (hunk 1 above)
Comment (grolimur) [reply engines_json/dblp.json]: Done.

**Comment by egh** [comment]
looks good, thank you!

**Comment by egh** [comment]
http://egh.github.io/zotero-lookup-engines/dblp.html

**Pull request merged**